.github/workflows/make-release.yml

```diff
@@ -0,0 +1,120 @@
+name: 'PyCyphal application builder'
+
+# Builds OpenCyphal PyCyphal release
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
Let us please follow the same policy here as in Yakut.

Reply:
I have separated the test and deployment into two separate workflows. The test workflow will run in all `push` and `pull_request` events. The deployment workflow will run on a push to `master` branch or a push of the `#release` tag to any branch.